addon/webextension/icons/help-16.svg

```diff
@@ -0,0 +1,6 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Swapped ? out for standard photon SVG

Reply:
Derp, wish I'd realized the circled '?' was a photon thing.